### PR TITLE
PriyaAssin9_selectors.html

### PR DESCRIPTION
**Pull request by priyaughade**

### CODE REVIEW

File: priya/HTML/PriyaAssign9_selectors.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    </style>
+ </head>
+ <body>
+    <h2><center>Achievers</center></h2>
+    <div class="clsdiv">
+      <div>
+        <h2>
+          Mountineer <span>Poona maalvath</span>,19,is conquering stereotypes
+        </h2>
+        <div>
```
Comment (MKadeer17) [inline priya/HTML/PriyaAssign9_selectors.html:40]: make: **font-variant: small-caps in css**

File: priya/HTML/PriyaAssign9_selectors.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        "I felt i was in heaven " gushes poorna malavath over a video call from
+        the us where she studying .Her broad, toothy smile momentairly
+        disappears.she chokes but doesen't give into tears.
+      </div>
+      <div>
+        <h2>
+          Scientiest and inventors <span> Geetanjali rao</span>,15 kid of the
+          youngest 2020.
+        </h2>
+        <div>
```
Comment (MKadeer17) [inline priya/HTML/PriyaAssign9_selectors.html:54]: Same here

**Review by MKadeer17 — COMMENTED** [review]
add font-variant: small-caps

File: priya/HTML/PriyaAssign9_selectors.html (hunk 1 above)
Comment (priyaughade) [reply priya/HTML/PriyaAssign9_selectors.html:40]: ok

**Review by MKadeer17 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**